### PR TITLE
Mitra uxlocal - interface for managing local resources

### PR DESCRIPTION
**Pull request by mitra42**
**Description**

This provides support for managing locally downloaded resources in dweb-mirror - it adds an indicator on Tiles when the item is downloaded. 

**Technical**

Its dependent on most of the other PR's I've already submitted, actual changes are only 
Adding CrawlConfig.js; and patching into NavWrap and TileComponent

**Testing**
Live in dweb-mirror

**Evidence**

### CODE REVIEW

File: packages/ia-components/sandbox/details/CrawlConfig.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
+import ReactDOM from "react-dom";
+
+const debug = require('debug')('dweb-archive:CrawlConfig');
+const canonicaljson = require('@stratumn/canonicaljson');
```
Comment (iisa) [inline packages/ia-components/sandbox/details/CrawlConfig.js]: are these able to be imports?  or do they need require?

Comment (mitra42) [reply packages/ia-components/sandbox/details/CrawlConfig.js]: debug definately needs require (it has to be loaded before the debug statement) 
canonicaljson probably doesn't ... I'll make that change.

Comment (mitra42) [reply packages/ia-components/sandbox/details/CrawlConfig.js]: Done

**Comment by iisa** [comment]
@mitra ^^ conflicts.

**Comment by iisa** [comment]
@mitra, the commit history is not reflective of changes here as well. i would like to get this in via squashed merge but am unsure of how this will affect your other branches. perhaps we should coordinate merges

cc @shaneriley

**Comment by mitra42** [comment]
Most of that commit history is merges i.e. rebases, done, partly at your request, IN ORDER to make the actual difference simpler. but if you look at https://github.com/internetarchive/iaux/pull/168/files there are only 3 file changes, (an addition of CrawlConfig.jsx, a change to NavWrap to use it (the one you found in #163 on its own) and the small UI change that this PR is all about.

**Pull request merged**